app/controllers/works_controller.rb

```diff
@@ -298,7 +298,11 @@ def handle_no_changes_or_invalid
   end
 
   def invalid_for_whats_changing_only?
-    @work_form.errors.one? && @work_form.errors.first.attribute == :whats_changing
+    @work_form.errors.one? && @work_form.errors.first.attribute == :whats_changing && !attached_files_changed?
+  end
+
+  def attached_files_changed?
+    @content.content_files.any? { |content_file| content_file.file_type == 'attached' }
```

Comment:
What if there are files deleted? Renamed? Globus?

Reply:
Ah yeah, on it.